openbb_terminal/helper_funcs.py

```diff
@@ -1149,7 +1149,6 @@ def replace_user_timezone(user_tz: str) -> None:
     """
     if is_timezone_valid(user_tz):
         dotenv.set_key(USER_ENV_FILE, "OPENBB_TIMEZONE", user_tz)
-        os.environ["OPENBB_TIMEZONE"] = user_tz
```

Comment:
Is there a reason this PR removes my previous bug fix? The bug of TZ not getting updated right away is reintroduced with this change.
<img width="966" alt="Screen Shot 2022-10-19 at 7 10 56 AM" src="https://user-images.githubusercontent.com/72827203/196602867-fed4f3f3-4c9a-49d8-a2d1-f87480ed721b.png">

Reply:
I have seen that fixed it in local, didn't push it yet

Reply:
Are we going to undo it?